src/Nut/DebugServiceRegistration.php

```diff
@@ -30,27 +34,43 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $this->displayRegistration($output);
+        $this->displayRegistration($input, $output);
     }
 
     /**
+     * @param InputInterface  $input
      * @param OutputInterface $output
      */
-    private function displayRegistration(OutputInterface $output)
+    private function displayRegistration(InputInterface $input, OutputInterface $output)
     {
         $table = new Table($output);
+        $rightAligned = new TableStyle();
+        $rightAligned->setPadType(STR_PAD_LEFT);
+
         $table->setHeaders([
-            '#',
-            'Provider',
+            [new TableCell('Service Providers', ['colspan' => 2])],
+            ['Provider Class Name', 'Boot Order']
```

Comment:
Can `Boot Order` be changed to just `Order`? Order affects services registered, booted, event subscriptions, and controllers mounted.